### PR TITLE
Add back myself as a member

### PR DESCRIPTION
**Pull request by mcollina**
I'm back as a member 🎉 !

Closes https://github.com/nodejs/security-wg/issues/650

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ You can show your users you take security matters seriously and drive higher con
 * [sam-github](https://github.com/sam-github) - **Sam Roberts**
 * [SomeoneWeird](https://github.com/SomeoneWeird) - **Adam Brady**
 * [vdeturckheim](https://github.com/vdeturckheim) - **Vladimir de Turckheim**
+* [mcollina](https://github.com/mcollina) - **Matteo Collina**
```
Comment (lirantal) [inline README.md:89]: apologies on the nit pick, we order the users on alphabetical order of their github handles 😆

**Comment by lirantal** [comment]
LGTM

**Comment by sam-github** [comment]
@mcollina I invited you to H1, given you are re-joining, do you need onboarding?

**Comment by mcollina** [comment]
is this any different than Node.js core?

**Comment by sam-github** [comment]
Yes, check out:

https://github.com/nodejs/security-wg/blob/master/processes/wg_onboarding.md#setting-up-for-new-wg-members

https://github.com/nodejs/security-wg/blob/master/processes/wg_onboarding.md#setting-up-for-new-triage-team-members

**Pull request merged**